README.md

```diff
@@ -167,6 +167,32 @@ $bunny = new Client($configuration);
 $bunny->connect();
 ```
 
+### Always run moving parts in a fiber
```

Comment:
Since this is a necessary part of using the library (code can be broken in case of a non-Fiber context https://github.com/jakubkulhan/bunny/issues/191), should this chapter be a bit higher than it is now?